lib/player.rb

```diff
@@ -0,0 +1,10 @@
+class Player
+  attr_accessor :name, :next_move, :moves, :token
```

Comment:
Are you sure you need to allow writing and reading permissions to all the variables?